tespy/components/customs.py

```diff
@@ -0,0 +1,818 @@
+
+        - :func:`tespy.components.customs.orc_evaporator.additional_equations`
+
+    Inlets/Outlets
+
+        - in1, in2, in3 (index 1: hot side 1, index 2: hot side 2,
+        index 3: cold side)
+        - out1, out2, out3 (index 1: hot side 1, index 2: hot side 2,
+        index 3: cold side)
+
+    Image
+
+        .. image:: _images/orc_evaporator.svg
+           :scale: 100 %
+           :alt: alternative text
+           :align: center
+
+    Parameters
+    ----------
+    label : str
+        The label of the component.
+
+    design : list
+        List containing design parameters (stated as String).
+
+    offdesign : list
+        List containing offdesign parameters (stated as String).
+
+    design_path: str
+        Path to the components design case.
+
+    local_offdesign : boolean
+        Treat this component in offdesign mode in a design calculation.
+
+    local_design : boolean
+        Treat this component in design mode in an offdesign calculation.
+
+    char_warnings: boolean
+        Ignore warnings on default characteristics usage for this component.
+
+    printout: boolean
+        Include this component in the network's results printout.
+
+    Q : String/float/tespy.helpers.dc_cp
+        Heat transfer, :math:`Q/\text{W}`.
+
+    pr1 : String/float/tespy.helpers.dc_cp
+        Outlet to inlet pressure ratio at hot side 1, :math:`pr/1`.
+
+    pr2 : String/float/tespy.helpers.dc_cp
+        Outlet to inlet pressure ratio at hot side 2, :math:`pr/1`.
+
+    pr3 : String/float/tespy.helpers.dc_cp
+        Outlet to inlet pressure ratio at cold side, :math:`pr/1`.
+
+    zeta1 : str/float/tespy.helpers.dc_cp
+        Geometry independent friction coefficient at hot side,
+        :math:`\frac{\zeta}{D^4}/\frac{1}{\text{m}^4}`.
+
+    zeta2 : str/float/tespy.helpers.dc_cp
+        Geometry independent friction coefficient at cold side,
+        :math:`\frac{\zeta}{D^4}/\frac{1}{\text{m}^4}`.
+
+    zeta3 : str/float/tespy.helpers.dc_cp
+        Geometry independent friction coefficient at cold side,
+        :math:`\frac{\zeta}{D^4}/\frac{1}{\text{m}^4}`.
+
+    subcooling : bool
+        Enable/disable subcooling at oulet of the hot side 1,
+        default value: disabled.
+
+    overheating : bool
+        Enable/disable overheating at oulet of the cold side,
+        default value: disabled.
+
+    Note
+    ----
+    The ORC evaporator are countercurrent heat exchangers.
+    Equation kA do not work for directcurrent and crosscurrent
+    or combinations of different types.
+
+    Example
+    -------
+    A 2-phase geo-fluid is used as the heat source for evaporating
+    the working fluid. The evaporator is designed for calculate the
+    mass flow rate of the working fluid with known steam and
+    brine mass flow rate. From this, it is possible to calculate
+    the mass flow rate of the working fluid that is fully evaporated
+    through the ORC evaporator and its heat transfer coefficient.
```

Comment:
Heat transfer coefficient does not work at the moment. Maybe there is a way to calculate that value from measured data and implement an additional equation?

Reply:
Currently, the kA value cannot be calculated because the principle of the ORC evaporator is still not clear for me. But it will be added after getting enough design parameters on the special component.